frontend/package.json

```
@@ -10,11 +10,18 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@radix-ui/react-slot": "1.2.2",
+    "@radix-ui/react-alert-dialog": "^1.1.14",
+    "@radix-ui/react-checkbox": "^1.3.2",
+    "@radix-ui/react-dialog": "^1.1.14",
+    "@radix-ui/react-dropdown-menu": "^2.1.15",
+    "@radix-ui/react-select": "^2.2.5",
+    "@radix-ui/react-slot": "^1.2.2",
+    "@radix-ui/react-tabs": "^1.1.12",
     "@react-router/node": "7.5.3",
     "@react-router/serve": "7.5.3",
     "@sentry/browser": "9.19.0",
     "@sentry/react": "9.19.0",
+    "@tanstack/react-table": "^8.21.3",
```

```suggestion
    "@radix-ui/react-alert-dialog": "1.1.14",
    "@radix-ui/react-checkbox": "1.3.2",
    "@radix-ui/react-dialog": "1.1.14",
    "@radix-ui/react-dropdown-menu": "2.1.15",
    "@radix-ui/react-select": "2.2.5",
    "@radix-ui/react-slot": "1.2.2",
    "@radix-ui/react-tabs": "1.1.12",
    "@react-router/node": "7.5.3",
    "@react-router/serve": "7.5.3",
    "@sentry/browser": "9.19.0",
    "@sentry/react": "9.19.0",
    "@tanstack/react-table": "8.21.3",
```
please pin the version